ecosystem-workload.config.js

```diff
@@ -1,13 +1,21 @@
-module.exports = {
+'use strict'
+
+const config = {
   apps: [{
+    name: 'server',
+    script: './server.js',
+    instances: 1
+  }]
+}
+
+if (process.env.WORKLOAD_DISABLED !== 'True') {
```

Comment:
I would just check the env var is not `undefined`, that way we don't have to remember it has to be exactly `True` rather than something like `true` or `1`.

Reply:
Normally I'd agree, but this value is a stringified boolean from Python (via the `compose.py` script), so we're not setting it directly. It will be either `False` or `True`.

It's set here: https://github.com/elastic/apm-integration-testing/pull/490/files#diff-d0df5dd5f0019c3b878fde30f4b7341b